src/modules/checkIsValidUserInput.js

```
@@ -9,7 +9,25 @@
  * @return {boolean} - True if the user input is valid, false otherwise
  */
 function checkIsValidUserInput(userInput) {
-  /* Write your code here */
+  if (userInput[0] === '0') {
+    return false;
```

Issue: The function returns false for invalid input, but the requirements specify that it should throw an error with a descriptive message for each validation failure. Please update this to throw an error instead.